components/suggestion/switch_channel_provider.jsx

```diff
@@ -221,21 +218,18 @@ class SwitchChannelSuggestion extends Suggestion {
                 {...Suggestion.baseProps}
             >
                 {icon}
-                <div className={classNames('suggestion-list__ellipsis', 'suggestion-list__flex')}>
+                <div className='suggestion-list__ellipsis suggestion-list__flex'>
                     <span className='suggestion-list__main'>
                         {name}
-                        {
-                            (isPartOfOnlyOneTeam || channel.type === Constants.DM_CHANNEL) &&
-                            <span className='ml-2 suggestion-list__desc'>{description}</span>
-                        }
+                        {(isPartOfOnlyOneTeam || channel.type === Constants.DM_CHANNEL) &&
+                            (<span className='ml-2 suggestion-list__desc'>{description}</span>
+                            )}
```

Comment:
this is the right way to write that in our context. How your IDE does not allow that should be of no concern.

```suggestion
                        {(isPartOfOnlyOneTeam || channel.type === Constants.DM_CHANNEL) && (
                            <span className='ml-2 suggestion-list__desc'>{description}</span>
                        )}
```